bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -792,8 +746,8 @@ if do_injections:
     th = ['Dist'] + ['Eff. Dist. %s' % site for site in sites] +\
          ['GPS time'] +\
          ['Inj. m1', 'Inj. m2', 'Inj. Mc', 'Rec. m1', 'Rec. m2', 'Rec. Mc',\
-          'Inj. inc', 'Inj. RA', 'Inj. Dec', 'Rec. RA', 'Rec. Dec', 'SNR',\
-          'Chi^2', 'Bank veto', 'Auto veto', 'Null SNR'] +\
+          'Inj. inc', 'Inj. RA', 'Inj. Dec', 'SNR',\
+          'Chi^2', 'Null SNR'] +\
```

Comment:
Ah, and here rec RA and Dec are gone.  Yes, looks like this needs a check for internal consistency.

Reply:
Yes, I missed this one. It appears that in the template bank there are no `ra`and `dec`